source/gui/nvdaControls.py

```diff
@@ -355,3 +354,63 @@ def onSliderChar(self, evt):
 			evt.Skip()
 			return
 		self.SetValue(newValue)
+
+
+class TabbableScrolledPanel(scrolledpanel.ScrolledPanel):
+	"""
+	This class was created to ensure a ScrolledPanel scrolls to nested children of the panel when navigating
+	with tabs (#12224). A PR to wxPython implementing this fix can be tracked on
+	https://github.com/wxWidgets/Phoenix/pull/1950
+	"""
+	def GetChildRectRelativeToSelf(self, child: wx.Window) -> wx.Rect:
+		"""
+		window.GetRect returns the size of a window, and its position relative to its parent.
+		When calculating ScrollChildIntoView, the position relative to its parent is not relevant unless the
+		parent is the ScrolledPanel itself. Instead, calculate the position relative to scrolledPanel
+		"""
+		cr = child.GetScreenRect()
+		spr = self.GetScreenPosition()

```

Comment:
Can you use more descriptive names for these please. I assume they stand for `childRect` and `screenPositionRect`  or is it `scrollPanelRect`? I think it is actually a `wx.Point` that is returned from `getScreenPosition`?
See: https://github.com/wxWidgets/wxWidgets/blob/db3e54b3dc5a4af812dea6e3945a2339ab01cbe2/include/wx/window.h#L315

Reply:
Fixed this, was poorly mixing wxPython conventions